docs/tools/offline/cli-test-debug.md

```diff
@@ -79,6 +86,32 @@ Test Case:
     Path: .\TESTS\functional\test3
 ```
 
+And in case of icetea:
```

Comment:
What do you mean by "in case of icetea"?  Do you mean to 'to find the tests that are available' for it, as in the previous example?